driver-core/src/main/com/mongodb/operation/CreateUserOperation.java

```diff
@@ -46,7 +46,9 @@
  * An operation to create a user.
  *
  * @since 3.0
+ * @deprecated use {@link CommandWriteOperation} directly or the mongod shell helpers.
  */
+@Deprecated
```

Comment:
I opted to deprecate these old user operations - this is because they don't support setting mechanisms and they are only used in deprecated methods in `DB`.

Reply:
sgtm